README.md

```diff
@@ -523,21 +523,28 @@ can install some projects that depend on MATLAB, in particular:
  * The [WB-Toolbox](https://github.com/robotology/WB-Toolbox) Simulink toolbox,
  * The [whole-body-controllers](https://github.com/robotology/whole-body-controllers) Simulink-based balancing controllers.
 
-
 To use this software, you can simply enable its compilation using the `ROBOTOLOGY_USES_MATLAB` CMake option.
 Once this software has been compiled by the superbuild, you just need to add some directories of the robotology-superbuild install (typically `$ROBOTOLOGY_SUPERBUILD_ROOT/build/install`) to [the MATLAB path](https://www.mathworks.com/help/matlab/matlab_env/what-is-the-matlab-search-path.html).
 In particular you need to add to the MATLAB path the `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/mex` directory and all the subdirectories `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/WBToolbox`.
 
-As an example, you could add this line to your MATLAB script that uses the robotology-superbuild matlab software:
+#### Start MATLAB from the launcher or the application menu
+
+You could add this line to your MATLAB script that uses the robotology-superbuild matlab software, substituting `<ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX>` with the `install` folder inside the build directory of the superbuild:
+
 ~~~
-    addpath([getenv('ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX')  '/mex'])
-    addpath([getenv('ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX')  '/share/WBToolbox'])
+    addpath('<ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX>' '/mex'])
+    addpath('<ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX>' '/share/WBToolbox'])
+    addpath('<ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX>' '/share/WBToolbox/images'])
 ~~~
```

Comment:
Ok, I reviewed a bit the section, and I think we can just remove this `addpath` calls in this section (if someone is interested in those, they are already described in the previous section), and  just leave the docs on how to run `startup_robotology_superbuild.m` under "Start MATLAB from the launcher or the application menu". This removes the ambiguity of using `ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX` env variable.

Reply:
Sorry, I did not realized that you stopped using `getenv`. Then I think we can go forward with the PR (as soon as @gabrielenava and @nunoguedelha green light it) and do more cleanup in separate PRs.